.taskcluster.yml

```diff
@@ -155,11 +154,62 @@ tasks:
         env: true
         events:
           - push
-          - create
         branches:
           - master
     metadata:
       name: Balrog Docker Image Creation
       description: Balrog Docker Image Creation
       owner: "{{ event.head.user.email }}"
       source: "{{ event.head.repo.url }}"
+
+# TODO: make this depend on the test task after https://bugzilla.mozilla.org/show_bug.cgi?id=1252144 is fixed
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - secrets:get:repo:github.com/mozilla/balrog:dockerhub
+    payload:
+      maxRunTime: 3600
+      image: "taskcluster/image_builder:0.1.3"
+      features:
+        dind: true
+        taskclusterProxy: true
+      command:
+        - "/bin/bash"
+        - "-c"
+        - "git clone {{ event.head.repo.url }} && cd balrog && git checkout {{ event.version }} && scripts/push-dockerimage.sh"
```

Comment:
Correct clone in this case will be: git clone -b {{ event.version }} {{ event.head.repo.url }} repo